Assets/PatchKit/UnityEditorExtension/Editor/UI/CreateAppScreen.cs

```diff
@@ -42,16 +42,50 @@ public override void UpdateIfActive()
 
     public override void Draw()
     {
-        GUILayout.Label(
-            string.Format(
-                "Create new PatchKit app for {0}",
-                _platform.ToDisplayString()),
-            EditorStyles.boldLabel);
-
-        _name = EditorGUILayout.TextField("Name: ", _name);
-
-        EditorGUILayout.Space();
+        if (GUILayout.Button(new GUIContent(_arrowIcon, "Change application"), GUILayout.Width(35), GUILayout.Height(20)))

```

Comment:
As single screen can be opened from any other, it can't display "Change application" as tooltip for going backwards.

Reply:
oversight, i will delete this tooltip